appconfig/settings/tailer.q

```diff
@@ -3,8 +3,9 @@
 \d .wdb
 savedir:hsym `$getenv[`KDBWDB]		// location to save wdb data
 hdbdir:hsym`$getenv[`KDBHDB]            // move wdb database to different location
-sortworkertypes:()			// tailer doesn't need to connect to sortworkers
-
+sortworkertypes:()			// WDB doesn't need to connect to sortworkers
```

Comment:
seems like a regression here - before the comment (correctly) called this tailer, now it says WDB?

Reply:
Apologies, this was the result of resolving a merge conflict incorrectly